### PR TITLE
Cache user avatar without '_dc' params

### LINKED ISSUE #274: User Profile Avatar not updating immediately after changing profile
When I change profile pic it is not directly update on top left thumbnail. When I change status then it's getting updated. And after that when i refresh it's changed back to previous one. I have attached gif for the same

Screen-Recording-2020-05-11-at-1

### PR DESCRIPTION
**Pull request by shubhsherl**
Closes #274 

This PR solves the update of the avatar. Except for the top thumbnail, which somehow shows a blank box after new avatar. 
While testing this PR, please wait for a few seconds before refreshing, to avoid [this](https://drive.google.com/file/d/17l_clh6-3BxPcVx-zLNQ5RiIDh-c73UD/view?usp=sharing) issue.

### CODE REVIEW

**Comment by ear-dev** [comment]
@Shailesh351 can you review this one too please?  Thanks!

**Comment by Shailesh351** [comment]
Will do @ear-dev 👍

**Comment by Shailesh351** [comment]
@shubhsherl  Changes looks good. 

For the 
> Except for the top thumbnail, which somehow shows a blank box after new avatar.

You can do following changes to fix it.
In the `updateAvatarOfUsername` function of `/app/ui-utils/client/lib/avatar.js`
Add this
```
// force reload of avatar on sidenav header
$(`.sidebar__header .sidebar__header-thumb .avatar-image[alt='${ username }']`)
		.attr('src', url);
```

for me it's working.

Thanks 😄

File: public/sw.js [hunk 1]
```diff
@@ -37,6 +43,12 @@ const fetchFromNetwork = (event) => {
 					}
 				})));
 			}
+
+			if (/avatar\/.*\?_dc/.test(event.request.url)) {
+				// handle the avatar updates
+				handleAvatar(event.request, response);
+			}
+
 			caches.open(version).then((cache) => cache.put(event.request, clonedResponse));
```
Comment (Shailesh351) [inline public/sw.js:52]: @shubhsherl without return statement in the if it will put in cache twice? Please confirm

Comment (shubhsherl) [reply public/sw.js:52]: It is required to update the avatars. RC gets any avatar update, with a param `_dc` which immediately updates the avatar.
So if we don't cache it, we have refresh once to reflect the changes.

**Review by Shailesh351 — APPROVED** [review]
LGTM :tada:

**Pull request merged**